FASTDOOM/i_ibm.c

```diff
@@ -694,6 +959,57 @@ void I_ProcessPalette(byte *palette)
 }
 #endif
 
+#if defined(MODE_CGA136) || defined(MODE_VGA136)
+void I_ProcessPalette(byte *palette)
+{
+    int i, j;
+    byte *ptr = gammatable[usegamma];
+
+    for (i = 0; i < 14 * 256; i++)
+    {
+        int distance;
+
+        int r1, g1, b1;
+
+        int best_difference = MAXINT;
+
+        r1 = (int)ptr[*palette++];
+        g1 = (int)ptr[*palette++];
+        b1 = (int)ptr[*palette++];
+
+        for (j = 0; j < 256; j++)
+        {
+            int r2, g2, b2;
+            int cR, cG, cB;
+            int pos = j * 3;
+
+            r2 = (int)colors[pos];
+            cR = abs(r2 - r1);
+
+            g2 = (int)colors[pos + 1];
+            cG = abs(g2 - g1);
+
+            b2 = (int)colors[pos + 2];
+            cB = abs(b2 - b1);
+
+            distance = cR + cG + cB;
+
+            if (distance == 0)
+            {
+                lut136colors[i] = j;
+                break;
+            }
+
```

Comment:
I didn't copy `distance = I_SQRT(distance);` from the MODE_CGA16 implementation of `I_ProcessPalette()` because I think it isn't necessary.